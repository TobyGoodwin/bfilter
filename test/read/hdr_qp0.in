Subject: yes =?utf-8?q?Just_=C2=A35!?= this is valid
Subject: no =?utf-8?q?Just_=A35!? not this
Subject: nope =?utf-9?q?Just_=A35!?= nor this
Subject: nada =?utf-8?z?Just_=A35!?= here either
Subject: empty =?iso-8859-1?q??=
Subject: two =?iso-8859-1?q?sep=61rate?= =?utf-8?q?pi=65ces?= of qp

this is the body
