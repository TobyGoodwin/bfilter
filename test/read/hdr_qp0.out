{ yes Just Â£5! this is valid}
{ no =?utf-8?q?Just_=A35!? not this}
{ nope =?utf-9?q?Just_=A35!?= nor this}
{ nada =?utf-8?z?Just_=A35!?= here either}
{ empty }
{ two separate pieces of qp}
{this is the body}
